### PR TITLE
Added support for google_mobile_ads 2.0.0

### PR DESCRIPTION
**Pull request by rmahmadkhan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,9 +43,11 @@ Update your app's `ios/Runner/Info.plist` file to add two keys:
 * A `GADApplicationIdentifier` key with a string value of your AdMob app ID ([identified in the AdMob UI](https://support.google.com/admob/answer/7356431)).
 * A `SKAdNetworkItems` key with Google's `SKAdNetworkIdentifier` value of `cstr6suwn9.skadnetwork` and for applovin as well
 
-**Note**
-The keys for AppLovin and Google Ads **are required** in Info.plist
-If you're not using any provider, write the line as it is otherwise provide your keys
```
Comment (nooralibutt) [inline README.md:48]: @rmahmadkhan did you test the example app on both android and ios platforms?

File: README.md [hunk 2]
```diff
@@ -190,9 +192,9 @@ class TestAdIdManager extends IAdIdManager {
     appId: Platform.isAndroid
         ? 'ca-app-pub-3940256099942544~3347511713'
         : 'ca-app-pub-3940256099942544~1458002511',
-    bannerId: BannerAd.testAdUnitId,
-    interstitialId: InterstitialAd.testAdUnitId,
-    rewardedId: RewardedAd.testAdUnitId,
+    bannerId: 'ca-app-pub-3940256099942544/6300978111',
+    interstitialId: 'ca-app-pub-3940256099942544/1033173712',
+    rewardedId: 'ca-app-pub-3940256099942544/5224354917',
```
Comment (nooralibutt) [inline README.md:197]: Why did u remove test ad ids? Are they no longer supported?

**Review by nooralibutt — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rmahmadkhan) [reply README.md:197]: Yeah. Latest version removes static testAdUnitId parameters. See the [Admob](https://developers.google.com/admob/flutter/test-ads) and [AdManager](https://developers.google.com/ad-manager/mobile-ads-sdk/flutter/test-ads) documentation for up to date test ad units.

File: README.md (hunk 1 above)
Comment (rmahmadkhan) [reply README.md:48]: Yes. Working fine on both android & iOS.

**Review by nooralibutt — APPROVED** [review]

**Pull request merged**